hypha.php

```diff
@@ -420,7 +418,7 @@ function data($name) {
 		switch ($name) {
 			//START_OF_DATA
 			case 'index': return '';
-			case 'system/core/language.php': return "json_decode('{\"en\": \"English\"}', true);";
+			case 'system/languages/languages.json': return json_decode(file_get_contents('system/languages/languages.json'), true);
```

Comment:
This now returns the decoded json (e.g. a PHP array), whereas the version that is generated when packing up hypha has this section replaced with actual file data, so that changes the return value. IOW, when you pack up hypha, the code no longer works.

To fix that, this should just return the result of `file_get_contents()`, and the `json_decode()` should be moved to the caller of `data()` elsewhere. Might be good to also test this packing up (though I'm not 100% sure it is working in master).

Reply:
fixed